src/snowflake/snowpark/functions.py

```diff
@@ -1138,7 +1136,7 @@ def approx_percentile_combine(state: ColumnOrName) -> Column:
     return builtin("approx_percentile_combine")(c)
 
 
-def explode(col: ColumnOrName) -> TableFunctionCall:
+def explode(col: ColumnOrName) -> "snowflake.snowpark.table_function.TableFunctionCall":
```

Comment:
Curious: why do we need this type hint change?

Reply:
The import statement on line 6-7 didn't fit well with other imports in the file and when I checked for the appropriate location, I found out that there was already an import use by `def table_function()` which uses type hints like the ones updated here. So I made the change to make them uniform across the file.